lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -976,6 +977,18 @@ SubExprInfo ExprEmitter::visitComb(ConstantOp op) {
   return {Unary, signPreference == RequireSigned ? IsSigned : IsUnsigned};
 }
 
+SubExprInfo ExprEmitter::visitComb(ArraySliceOp op) {
+  auto arrayPrec = emitSubExpr(op.input(), Symbol);
+
+  unsigned dstWidth = op.getType().cast<ArrayType>().getSize();
+  os << '[';
+  emitSubExpr(op.lowIndex(), LowestPrecedence);
```

Comment:
@lattner Is this the correct way to emit two identical expressions on the same line?